src/sonic-bgpcfgd/bgpcfgd/manager_static_route.py

```diff
@@ -0,0 +1,173 @@
+import traceback
```

Comment:
Please rename this file to `managers_static_rt.py` to be consistent with existing files

Reply:
Renamed the file name to `managers_static_rt.py`.